acceptance/bundle/deploy/files/no-snapshot-sync/script

```diff
@@ -0,0 +1,40 @@
+envsubst < databricks.yml.tmpl > databricks.yml
+
+cleanup() {
+    trace $CLI bundle destroy --auto-approve
+}
+trap cleanup EXIT
+
+trace $CLI bundle deploy
+
+title "Check that test file is in workspace"
+trace $CLI workspace get-status "/Workspace/Users/${CURRENT_USER_NAME}/.bundle/${UNIQUE_NAME}/files/test.py" | jq '{object_type,path}'
+trace $CLI workspace get-status "/Workspace/Users/${CURRENT_USER_NAME}/.bundle/${UNIQUE_NAME}/files/test_to_modify.py" | jq '{object_type,path}'
+
+title "Check that notebook is in workspace"
+trace $CLI workspace get-status "/Workspace/Users/${CURRENT_USER_NAME}/.bundle/${UNIQUE_NAME}/files/notebook" | jq '{object_type,path,language}'
+
+title "Check that deployment.json is synced correctly"
+trace $CLI workspace get-status "/Workspace/Users/${CURRENT_USER_NAME}/.bundle/${UNIQUE_NAME}/state/deployment.json" | jq '{object_type,path}'
+
+title "Remove .databricks directory to simulate a fresh deployment like in CI/CD environment"
+rm -rf .databricks
+
+title "Remove the file from the bundle"
+rm test.py
```

Comment:
nit: "trace rm test.py" is shorter and more descriptive than title.

Reply:
did some regroup in [c5e49e3](https://github.com/databricks/cli/pull/2924/commits/c5e49e3143d1ec69af124976774ceef6e1319705)